File: Oxiyopal.Models/Ubicacion.cs
using Oxiyopal.Models.Base;
using Oxiyopal.Models.ViewModels;
using System;
using System.Collections.Generic;
using System.Text;

namespace Oxiyopal.Models
{
    public class Ubicacion : EntityBase<int>
    {
        public bool estaEnBodega { get; set; }


        public int? ClienteId { get; set; }
        public Cliente Cliente { get; set; }
        public int? BodegaId { get; set; }
        public Bodega Bodega { get; set; }
        public IList<HistorialCilindro> HistorialCilindros { get; set; }

       
    }
}

File: Oxiyopal.WebUI/ClientApp/src/app/models/Ubicacion.ts
import { EntityBase } from './Base/EntityBase';
import { HistorialCilindro } from './ViewModels/HistorialCilindro';
import { Cliente } from './Cliente';
import { Bodega } from './Bodega';

export interface Ubicacion extends EntityBase<number> {
    estaEnBodega: boolean;
    fechaDeTraslado: Date | string;

    clienteId: number;
    cliente: Cliente;
    bodegaId: number;
    bodega: Bodega;
    historialCilindros: HistorialCilindro[];
}

